cdk/package.json

```diff
@@ -18,15 +18,13 @@
     "diff": "cdk diff --path-metadata false --version-reporting false"
   },
   "devDependencies": {
-    "@aws-cdk/aws-apigatewayv2": "1.113.0",
-    "@aws-cdk/aws-apigatewayv2-alpha": "^2.14.0-alpha.0",
     "@guardian/cdk": "v59.3.1",
     "@guardian/eslint-config-typescript": "1.0.11",
     "@guardian/prettier": "2.1.5",
     "@types/jest": "^29.5.12",
-    "@types/node": "20.13.0",
-    "aws-cdk": "2.122.0",
-    "aws-cdk-lib": "2.130.0",
+    "@types/node": "22.5.3",
```

Comment:
Does this need to match the node version we run?

Reply:
Can you see docs to say this tracks? It looks like it does, but I can't find any evidence that it does, [bar on stackoverflow which suggests it didn't, then it did, then it didn't again](https://stackoverflow.com/questions/42035263/relationship-between-the-version-of-node-js-and-the-version-of-types-node)?